server/endpoint/status/endpoint.go

```diff
@@ -99,6 +109,12 @@ func (e Endpoint) Endpoint() kitendpoint.Endpoint {
 			return nil, microerror.Mask(err)
 		}
 
+		if !key.InCluster(*currentCR) {
+			if request.AuthToken != e.webhookAuthToken {
+				return nil, microerror.Maskf(wrongTokenError, "got wrong auth token %q for %#q app %#q namespace", request.AuthToken, request.AppName, request.AppNamespace)
```

Comment:
```suggestion
				return nil, microerror.Maskf(wrongTokenError, "incorrect auth token for %#q app %#q in namespace", request.AppName, request.AppNamespace)
```

I'm being paranoid as usual :smile: but I'd prefer not to log the token. If there is a misconfiguration then we need to reset the token as it can be in logs.

I think this is still clear and OK for debugging. WDYT?

Reply:
You don't want to print the wrong token, too? I see It's not giving any value I think.